src/elements/pages/dev/game/game-banner.ts

```diff
@@ -24,10 +25,22 @@ export default class DevGameBanner extends LitElement {
 
 	render() {
 		return html`
-			<div class="relative my-[20px] md:my-[40px] mx-auto bg-zinc-700 h-[18.2rem] rounded-[20px]">
+			<div
+				class=${clsx(
+					'relative my-[20px] md:my-[40px] mx-auto h-[18.2rem] rounded-[20px]',
```

Comment:
Same as pervious PR - consider using close enough variables from Tailwind theme instead of introducing custom ones. In case of any problem, just let me know, I'm happy to help.

Reply:
Yeah, I fixed them in the other PR. Chose not to do it again here just because it would lead to inconsistent styling since this PR doesn't have the tailwind-based userBanner and because there was already an open PR that addressed that issue.